pkg/util/grpcclientutil/grpcclientutil.go

```diff
@@ -42,12 +42,12 @@ func ConsumeInterfaces(netVariables object.NetVariables) ([]*Interface, error) {
 }
 
 func getInterfaceServiceClient() (InterfaceServiceClient, *grpc.ClientConn, context.Context, context.CancelFunc, error) {
-	conn, err := grpc.Dial("localhost:50051", grpc.WithInsecure(), grpc.WithBlock())
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*8)
```

Comment:
Please change the timeout to "time.Second * 2" and make it a constant at the top of this file or if we have a place for go consts.